pkg/model/crd.go

```diff
@@ -387,6 +388,81 @@ func (r *CRD) GetOutputWrapperFieldPath(
 	return &opConfig.OutputWrapperFieldPath
 }
 
+// GetOutputShape returns the Output shape for given operation.
+func (r *CRD) GetOutputShape(
+	// The operation to look for the Output shape
+	op *awssdkmodel.Operation,
+) (*awssdkmodel.Shape, error) {
+	if op == nil {
+		return nil, nil
+	}
+
+	outputShape := op.OutputRef.Shape
+	if outputShape == nil {
+		return nil, errors.New("output shape not found")
+	}
+
+	// We might be in a "wrapper" shape. Unwrap it to find the real object
+	// representation for the CRD's createOp/DescribeOP.
+
+	// Use the wrapper field path if it's given in the ack-generate config file.
+	wrapperFieldPath := r.GetOutputWrapperFieldPath(op)
+	if wrapperFieldPath != nil {
+		wrapperOutputShape, err := r.GetWrapperOutputShape(outputShape, *wrapperFieldPath)
+		if err != nil {
+			return nil, err
```

Comment:
```suggestion
			return nil, fmt.Errorf("unable to unwrap the output shape: %v", err)
```

Reply:
Thanks for the comment, I have updated the code.